### PR TITLE
Add support for pagerfanta 3

### PR DESCRIPTION
**Pull request by loic425**
replaces #143

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -57,15 +57,15 @@
...
+        "pagerfanta/pagerfanta": "^3.0",
         "pamil/phpspec-skip-example-extension": "^4.2",
         "phpspec/phpspec": "^7.0",
         "phpstan/phpstan": "0.12.99",
         "phpstan/phpstan-phpunit": "0.12.22",
         "phpstan/phpstan-webmozart-assert": "0.12.16",
         "phpunit/phpunit": "^9.5",
         "sylius-labs/coding-standard": "^4.0",
-        "sylius/resource-bundle": "^1.8",
+        "sylius/resource-bundle": "^1.9",
```
Comment (lchrusciel) [inline composer.json:68]:
```suggestion
        "sylius/resource-bundle": "^1.9@alpha",
```

File: composer.json [hunk 2]
```diff
@@ -120,5 +120,7 @@
         "test-with-postgres": [
             "APP_ENV=test_with_postgres vendor/bin/phpunit --colors=always"
         ]
-    }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true
```
Comment (lchrusciel) [inline composer.json:125]: I'm not sure if prefer stable should stay, but we may leave it for now
```suggestion
    },
    "prefer-stable": true
```

**Comment by loic425** [comment]
@lchrusciel I've applied your suggestions and CI is green.

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, Loïc! :1st_place_medal:

**Pull request merged**